src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperCompletionService.cs

```diff
@@ -21,6 +22,99 @@ public DefaultTagHelperCompletionService(TagHelperFactsService tagHelperFactsSer
             _tagHelperFactsService = tagHelperFactsService;
         }
 
+        public override AttributeCompletionResult GetAttributeCompletions(AttributeCompletionContext completionContext)
```

Comment:
This could use a comment explaining at a high level how/why we expect this to behave.